### DISCUSSION TITLE
Dumper for np.float64 or np.int64

### DISCUSSION THREAD

**Discussion by ghost** [psycopg/psycopg#404] [Q&A]
Hi,
i would like to dump np.float64 or np.int64. 
How cas i manage to perform insert/update (pyscopg3) ?
Best regards.

**Comment by dvarrazzo** [comment]
There is a MR in #332 by @vertefra to address this feature.

I should actually review his work. But in the meantime there is already something you can use there.

Feel free to take a look and let us know if you need further help.

**Reply by ghost** [reply]
Many thanks
